test/jdk/tools/jmod/hashes/HashesTest.java

```diff
@@ -401,19 +378,16 @@ private ModuleHashes hashes(String name) {
     private ModuleHashes hashes(ModuleFinder finder, String name) {
         ModuleReference mref = finder.find(name).orElseThrow(RuntimeException::new);
         try {
-            ModuleReader reader = mref.open();
-            try (InputStream in = reader.open("module-info.class").get()) {
+            try (ModuleReader reader = mref.open(); InputStream in = reader.open("module-info.class").get()) {
                 ModuleHashes hashes = ModuleInfo.read(in, null).recordedHashes();
                 System.out.format("hashes in module %s %s%n", name,
-                    (hashes != null) ? "present" : "absent");
+                        (hashes != null) ? "present" : "absent");
                 if (hashes != null) {
                     hashes.names().stream().sorted().forEach(n ->
-                        System.out.format("  %s %s%n", n, toHex(hashes.hashFor(n)))
+                            System.out.format("  %s %s%n", n, toHex(hashes.hashFor(n)))
```

Comment:
Nit: the extra whitespaces in line 384 and 387 may be added by IDE.  Can you revert it.

Reply:
Intellij did that.  I can tweak